requirements-dev.txt

```diff
@@ -0,0 +1,11 @@
+certifi==2023.11.17
+charset-normalizer==3.3.2
+idna==3.6
+numpy==1.26.2
+pandas==2.1.4
+python-dateutil==2.8.2
+pytz==2023.3.post1
+requests==2.31.0
+six==1.16.0
+tzdata==2023.3
+urllib3==2.1.0
```

Comment:
Instead of auto-generating the requirements file, can you only add direct project dependencies please?